torchmetrics/functional/regression/explained_variance.py

```diff
@@ -40,7 +40,7 @@ def _explained_variance_compute(
     sum_target: Tensor,
     sum_squared_target: Tensor,
     multioutput: str = "uniform_average",
-) -> Union[Tensor, Sequence[Tensor]]:
+) -> Tensor:
```

Comment:
This does not seem correct to me. Even when setting `multioutput='raw_values` it is still a single tensor that is returned

Reply:
we return the Tensor now or am I missing anything?